### DISCUSSION TITLE
[SOLVED]local zone DNS in Adguard Home RPI

### DISCUSSION THREAD

**Discussion by 3lliryc** [AdguardTeam/AdGuardHome#5341] [Q&A]
hi,
i use adguard home recently on a raspbery pi 4.
I need to create dns entries for my local network:
i used this feature in pi-hole.

example:
192.168.10.20 nas.localdomain.lan
192.168.10.10 intranet.localdomain.lan

etc.

Is the feature is available on adguard home?
i don't find on GUI admin.

Best,

**Comment by ppfeufer** [comment]
Multiple solutions:

Menu:
Filters -> DNS Rewrites

Custom Rules:
Filters -> Custom filtering rules -> Add the rule here (For syntax see https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#dnsrewrite)

The custom rules for your examples would look like:
```text
|nas.localdomain.lan^$dnsrewrite=NOERROR;A;192.168.10.20
|intranet.localdomain.lan^$dnsrewrite=NOERROR;A;192.168.10.10
```

**Reply by 3lliryc** [reply]
thanks a lot.

**Reply by ppfeufer** [reply]
If this solves your little problem, please mark this thread as answered :-)
